lookup.js

```diff
@@ -91,15 +92,15 @@ exports.init = function (sbot, config) {
     }
   }
 
-  function msgToDetails(prevDetails, msg) {
+  function msgToDetails(msg) {
     const content = msg.value.content
-    const details = { ...prevDetails }
-    details.id = prevDetails ? prevDetails.id : content.subfeed
-    details.parent = content.metafeed || details.parent
-    details.purpose = content.feedpurpose || details.purpose
+    const details = {}
+    if (content.subfeed) details.id = content.subfeed
+    if (content.metafeed) details.parent = content.metafeed
+    if (content.feedpurpose) details.purpose = content.feedpurpose
     details.feedFormat = validate.detectFeedFormat(content.subfeed)
     details.recps = content.recps || null
-    details.metadata = {} || details.metafeed
+    details.metadata = {} || details.metadata
```

Comment:
here details.metadata will always be undefined because you start details empty

Reply:
Oh shoot